src/scss/custom/components/customer/_customer-menu.scss

```diff
@@ -24,7 +24,7 @@ $componentName: account-menu;
     }
 
     &.#{$componentName}--signout {
-      .#{$componentName}__link {
+      .link-item {
```

Comment:
I think this is pretty bad honestly, we're going against our practices in order to unify with old modules that should apply to our naming, don't you think?

Overwise we would have kept every old classes :/

Reply:
@NeOMakinG I renamed to the old one because `.#{$componentName}__link` is confusing, it's not a link.

Can we name it `.#{$componentName}__link-content`?

Reply:
I think so, no problem with that